src/sentry/static/sentry/app/views/organizationGroupDetails/header.tsx

```diff
@@ -38,27 +41,36 @@ const TAB = {
   SIMILAR_ISSUES: 'similar-issues',
 };
 
-class GroupHeader extends React.Component {
-  static propTypes = {
-    currentTab: PropTypes.string.isRequired,
-    baseUrl: PropTypes.string.isRequired,
-    group: SentryTypes.Group.isRequired,
-    project: SentryTypes.Project,
-    api: PropTypes.object,
-  };
+type Props = {
+  currentTab: string;
+  baseUrl: string;
+  group: Group;
+  project: Project;
+  api: Client;
+};
 
+type MemberList = NonNullable<
+  React.ComponentProps<typeof AssigneeSelector>['memberList']
+>;
+
+type State = {
+  memberList?: MemberList;
+};
+
+class GroupHeader extends React.Component<Props, State> {
   static contextTypes = {
     location: PropTypes.object,
     organization: SentryTypes.Organization,
   };
 
-  state = {memberList: null};
+  state: State = {};
 
   componentDidMount() {
     const {organization} = this.context;
-    const {project} = this.props.group;
+    const {group} = this.props;
+    const {project} = group;
 
-    fetchOrgMembers(this.props.api, organization.slug, project.id).then(memberList => {
+    fetchOrgMembers(this.props.api, organization.slug, [project.id]).then(memberList => {
```

Comment:
Should `api` be destructured together with `group`?

Reply:
sure thing